garak/generators/nvcf.py

```diff
@@ -55,6 +55,21 @@ def __init__(self, name=None, generations=10):
             "Accept": "application/json",
         }
 
+    def _build_payload(self, prompt) -> dict:
+
+        payload = {
+            "messages": [{"content": prompt, "role": "user"}],
+            "temperature": self.temperature,
+            "top_p": self.top_p,
+            "max_tokens": self.max_tokens,
+            "stream": False,
+        }
+
+        return payload
+
+    def _extract_text_output(self, response) -> str:
+        return response["choices"][0]["message"]["content"]
```

Comment:
Consider adjusting to have the `List` wrapper in the return signature here and avoiding need to `wrap` in `_call_model` returns. Same for the `NvcfCompletion` implementation.

This seems similar to `openai.py` consider: (syntax may differ for access to dict vs object members though)
```suggestion
    def _extract_text_output(self, response) -> List[str]:
        return [c.message.content for c in response.choices]
```